render-only-sample/rosumd/RosUmdResource.h

```diff
@@ -121,6 +136,12 @@ class RosUmdResource : public RosAllocationExchange
         UINT rowStride, 
         BOOLEAN OddRow);
 
+    void CalculateTilesInfo();
+
+    void FillTileInfo(UINT bpp);
```

Comment:
Prefer pure functions where possible - a pure function is a function where the return value is only determined by its input values, without observable side effects. It doesn't look like FillFileInfo needs any members other than m_TileInfo.  Consider if this logic may be better suited for a constructor of VC4TileInfo or static method that returns a VC4TileInfo structure by value. (Return value optimization will ensure that structure does not actually get copied).